Console/Helpers/DummyData/AdapterDummy.cs

```diff
@@ -1,56 +1,55 @@
 ﻿using System.Collections.Generic;
-using Wokhan.WindowsFirewallNotifier.Console.Helpers.ViewModels;
 
 namespace Wokhan.WindowsFirewallNotifier.Console.Helpers.DummyData
 {
     public class AdapterDummy
     {
         public class IPInterfaceStatistics
         {
-            public long BytesReceived { get { return 10000; } }
-            public long BytesSent { get { return 10000; } }
-            public long IncomingPacketsDiscarded { get { return 10000; } }
-            public long IncomingPacketsWithErrors { get { return 10000; } }
-            public long IncomingUnknownProtocolPackets { get { return 10000; } }
-            public long NonUnicastPacketsReceived { get { return 10000; } }
-            public long NonUnicastPacketsSent { get { return 10000; } }
-            public long OutgoingPacketsDiscarded { get { return 10000; } }
-            public long OutgoingPacketsWithErrors { get { return 10000; } }
-            public long OutputQueueLength { get { return 10000; } }
-            public long UnicastPacketsReceived { get { return 10000; } }
-            public long UnicastPacketsSent { get { return 10000; } }
+            public long BytesReceived => 10000;
+            public long BytesSent => 10000;
+            public long IncomingPacketsDiscarded => 10000;
+            public long IncomingPacketsWithErrors => 10000;
+            public long IncomingUnknownProtocolPackets => 10000;
+            public long NonUnicastPacketsReceived => 10000;
+            public long NonUnicastPacketsSent => 10000;
+            public long OutgoingPacketsDiscarded => 10000;
+            public long OutgoingPacketsWithErrors => 10000;
+            public long OutputQueueLength => 10000;
+            public long UnicastPacketsReceived => 10000;
+            public long UnicastPacketsSent => 10000;
         }
 
         public class IPInterfaceProperties
         {
-            public List<string> AnycastAddresses { get { return new List<string> { "12345" }; } }
-            public List<string> DhcpServerAddresses { get { return new List<string> { "12345" }; } }
-            public List<string> DnsAddresses { get { return new List<string> { "12345" }; } }
-            public string DnsSuffix { get { return ""; } }
-            public List<string> GatewayAddresses { get { return new List<string> { "12345" }; } }
-            public bool IsDnsEnabled { get { return true; } }
-            public bool IsDynamicDnsEnabled { get { return false;  } }
-            public List<string> MulticastAddresses { get { return new List<string> { "12345" }; } }
-            public List<string> UnicastAddresses { get { return new List<string> { "12345" }; } }
-            public List<string> WinsServersAddresses { get { return new List<string> { "12345" }; } }
+            public List<string> AnycastAddresses => new List<string> { "12345" };

```

Comment:
Interesting...

Reply:
I don't remember removing MulticastAdresses and the like.... did I really do that?!
Or maybe you were only mentionning the => notation (shortcut for getters).